remote/bitbucketserver/helper.go

```diff
@@ -0,0 +1,64 @@
+package bitbucketserver
+
+import (
+	"net/http"
+	"bytes"
+	"io/ioutil"
+	"fmt"
+	"strings"
+	"crypto/md5"
+	log "github.com/Sirupsen/logrus"
+	"encoding/hex"
+)
+
+func avatarLink(email string) (url string) {
+	hasher := md5.New()
+	hasher.Write([]byte(strings.ToLower(email)))
+	emailHash := fmt.Sprintf("%v", hex.EncodeToString(hasher.Sum(nil)))
+	avatarURL := fmt.Sprintf("http://www.gravatar.com/avatar/%s.jpg",emailHash)
+	log.Info(avatarURL)
```

Comment:
we should use `https` instead of `http`, otherwise if you use `https` you will get a browser error due to insecure content